validator/client/validator.go

```diff
@@ -939,8 +940,14 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 
 // UpdateFeeRecipient calls the prepareBeaconProposer RPC to set the fee recipient.
 func (v *validator) UpdateFeeRecipient(ctx context.Context, km keymanager.IKeymanager) error {
+	// only used after Bellatrix
 	if v.feeRecipientConfig == nil {
-		log.Warnln("Fee recipient config not set, skipping fee recipient update. Validator will continue proposing using beacon node specified fee recipient.")
+		e := params.BeaconConfig().BellatrixForkEpoch
+		if e != math.MaxUint64 && slots.ToEpoch(slots.CurrentSlot(v.genesisTime)) < e {
+			log.Warnln("Please plan for using the fee recipient flags post Bellatrix Hard Fork to receive transaction fee rewards on validator work!!")
```

Comment:
Is this logrus or the stdlib's log package? With logrus we typically just do log.Warn("foo") nor Warnln.

Here I would also recommend we use crystal clear language assuming a user has never heard about this feature. Some ideas:

`After the Ethereum merge, you will need to specify the Ethereum addresses which will receive transaction fees from proposing blocks. This is known as a fee recipient configuration. You can read more about this feature in our documentation portal here <LINK>`

Reply:
Let's avoid exclamation marks for consistency